docs/spanish.md

```diff
@@ -24,27 +25,54 @@ Some terms are very tricky to translate, so be sure to communicate
 with other Spanish speakers in the community. It's all about making
 Zulip friendly and usable.
 
-## Términos Especiales Usados en Zulip
-
-* mention - **@mención**
-* customization - **personalización**
-* view / filter - **vista**
-* PM / Private Message - **MP grupal**
-* Home - **Inicio**: We never use the term "Hogar" (literally home) in Spanish.
-* message table - **tabla de mensajes**
-* muting a stream - **silenciar un canal**
-* narrow - **filtrar**: there's no other word that's common enough in
-  Spanish for "to narrow" except for "filtrar". This is why we'll use
-  "filtrar" for "to narrow" and not for the english Zulip meaning
-  "filter", fo which we'll use "view" instead
-* pinning - **fijar** (lit. to fixate)
-* stream - **canal** The use of the literal Spanish word for stream
-  "Flujo" is very confusing and not the correct metaphor for Spanish
-  speaking people. The correct term would be "canal" (channel).
-* realm - **instancia**: The literal Spanish of realm is "reino",
+## Términos
+* Message - **Mensaje**
+* Private message (PM) - **Mensaje privado (MP)**
+* Group PM - **MP grupal**: this is the most precise translation. However,
+  preferrably don't use this, since many users may not associate "MP" with
+  *private message* in a group context. Better use it without abbreviations
+  (**mensaje privado grupal**).
```

Comment:
I'd recommend phrasing this differently to not mention "MP grupal" in bold, since if not reading carefully, one would think that's the asnwer.